pyteal/ast/subroutine_test.py

```diff
@@ -212,20 +190,29 @@ def fnWithScratchVarSubclass(a, b: DynamicScratchVar):
         ),
         (
             fnWithNonExprReturnAnnotation,
-            "Function has return of disallowed type TealType.uint64. Only subtype of Expr is allowed",
+            "Function has return of disallowed type TealType.uint64. Only Expr is allowed",
         ),
         (
             fnWithNonExprParamAnnotation,
-            "Function has parameter b of disallowed type TealType.uint64. Only the types",
+            "Function has parameter b of declared type TealType.uint64 which is not a class",
         ),
         (
             fnWithScratchVarSubclass,
             "Function has parameter b of disallowed type <class 'pyteal.DynamicScratchVar'>",
         ),
+        (
+            fnReturningExprSubclass,
+            "Function has return of disallowed type <class 'pyteal.Return'>",
+        ),
+        (
+            fnWithMixedAnns4AndBytesReturn,
+            "Function has return of disallowed type <class 'pyteal.Bytes'>",
+        ),
     )
 
     for fn, msg in cases:
         with pytest.raises(TealInputError) as e:
+            print(f"case=[{msg}]")
```

Comment:
Is this print statement intentional?

Reply:
Yes. Instead of catching the exception and re-raising with the `case` as I was doing before, now I'm just printing out each case. So if you run `pytest -sv` you can see the last `case=XYZ` that got printed and infer the broken case.